tests/test_compat.py

```diff
@@ -0,0 +1,162 @@
+"""Tests for compatability functions."""
```

Comment:
can you also add tests for the directions of the basis vectors? IE, is this flipping the direction of zeta?